Update issue templates (#26)
datasciencecampus/pprl_toolkit: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: "[BUG] : "
labels: bug
assignees: ''

---

Please be aware that, as the `pprl_toolkit` is an experimental package, ONS cannot promise to resolve bugs.

> **Describe the bug**

Reviewer: ```suggestion
### Describe the bug
```

A clear and concise description of what the bug is.

> **To Reproduce**

Reviewer: ```suggestion
### How to reproduce the bug
```

> Steps to reproduce the behavior:

Reviewer: ```suggestion
Steps to reproduce the behaviour:
```

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

> **Expected behavior**

Reviewer: ```suggestion
### Expected behaviour
```

A clear and concise description of what you expected to happen.

> **Screenshots**

Reviewer: ```suggestion
### Evidence (tracebacks and screenshots
```

> If applicable, add screenshots to help explain your problem.

Reviewer: ```suggestion
If applicable, please add any tracebacks or screenshots to help explain your problem.
```

> **Desktop (please complete the following information):**
>  - OS: [e.g. iOS]
>  - Browser [e.g. chrome, safari]
>  - Version [e.g. 22]

Reviewer: ```suggestion
### System information
Please provide the following information about your environment:

 - OS: [e.g. macOS]
 - Browser (when using the client-side app or GCP): [e.g. Chrome, Safari]
 - `pprl_toolkit` version: [e.g. 0.0.1]
```

> **Additional context**

Reviewer: ```suggestion
### Additional context
```
